src/app/component/general/eco-news/eco-news-detail/eco-news-widget/eco-news-widget.component.html

```diff
@@ -0,0 +1,11 @@
+<div class=container>
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/77413852-b70a1700-6dc8-11ea-81c7-f17df55b1fc7.png)

Reply:
Please add this element to the layout

Reply:
fixed